### PR TITLE
Readme: create initial readme

### PR DESCRIPTION
**Pull request by RiRa12621**
This PR adds an initial readme. that contains the basic set of information about this repository as well as a slim set of guidelines how to contribute

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+shipping the respective alert. This results in a structure as follows:
+
+`root/alerts/operator_repository_name/some_alert.md`
+
+The root folder contains all information and files required for this repository
+in general, while all runbooks live in their respective subfolder.
+
+## Onboarding
+If your operator has not shipped any runbooks so far, please create a subfolder
+following the repository layout and place and OWNERS file in it. The OWNERS file
```
Comment (jan--f) [inline README.md:25]:
```suggestion
following the repository layout and place an OWNERS file in it. The OWNERS file
```

**Review by jan--f — CHANGES_REQUESTED** [review]
Just a typo, otherwise good start :+1:

**Comment by RiRa12621** [comment]
Thanks, fixed @jan--f . PTAL again

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# OpenShift Runbooks
```
Comment (NautiluX) [inline README.md:1]:
```suggestion
# OpenShift Runbooks

```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+A collection of runbooks for alert rules that are shipped with OpenShift.
+
+## How To Use
```
Comment (NautiluX) [inline README.md:4]:
```suggestion
## How To Use

```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+OpenShift Container Platform is shipping a finely tuned set of alerts to inform
+the cluster's owner and/or operator of events and bad conditions in the cluster.
+This repository is a consistently growing collection of runbooks for said
+alerts, that are intended to guide a cluster owner and/or operator through the
+steps of fixing problems on clusters, which are surfaced by alerts.
+
+Going forward alerts in OpenShift will ship a link to the corresponding runbook
```
Comment (NautiluX) [inline README.md:11]:
```suggestion
Going forward, alerts in OpenShift will ship a link to the corresponding runbook
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+in this repository, to make fixing problems even easier.
+
+## Repository Layout
```
Comment (NautiluX) [inline README.md:14]:
```suggestion
## Repository Layout

```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+Runbooks in this repository are grouped by the operator that is responsible for
```
Comment (NautiluX) [inline README.md:15]: are there only operators? Or should it be "component"

File: README.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+shipping the respective alert. This results in a structure as follows:
+
+`root/alerts/operator_repository_name/some_alert.md`
+
+The root folder contains all information and files required for this repository
+in general, while all runbooks live in their respective subfolder.
+
+## Onboarding
```
Comment (NautiluX) [inline README.md:23]:
```suggestion
## Onboarding

```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+If your operator has not shipped any runbooks so far, please create a subfolder
+following the repository layout and place an OWNERS file in it. The OWNERS file
+should be the same as the one in the operator's main repository. Additionally
+add all runbooks in said folder, that you want to contribute.
```
Comment (NautiluX) [inline README.md:27]:
```suggestion
add all runbooks to said folder that you want to contribute.
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+## Contributing
```
Comment (NautiluX) [inline README.md:29]:
```suggestion
## Contributing

```

**Review by NautiluX — APPROVED** [review]
looks good to me, only a couple of formatting suggestions.

File: README.md (hunk 6 above)
Comment (RiRa12621) [reply README.md:15]: I specifically used "operator" so people don't add one folder per subcomponent. EG all monitoring things are under cluster-monitoring-operstor and not Prometheus operator and alertmanager and Prometheus

**Pull request merged**